docker/docker-compose-ssl.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+services:
+  saltmaster-local-ssl:
```

Comment:
the minions are referring to "saltmaster-local", so they then cannot find this salt-master.